### ISSUE TITLE
[Customising web-ui] - Projects are loading tasks in web ui of self hosting server but i want them to show datasets 

### ISSUE THREAD

**Issue by sriram-dsl** [allegroai/clearml-server#224]
![image](https://github.com/allegroai/clearml-server/assets/137028273/b6b13c37-754b-4f6d-85f8-40004d592314)

projects are not loading correctly : 
I am facing an issue with the projects menu; it is not loading correctly. Despite the folders being in use, the information is not displayed in the projects menu. Interestingly, the data is visible in the datasets menu. All the folders contain data, which is visible in the datasets menu but not in the projects menu.

Additionally, if the projects menu could display running tasks and completed tasks, I'm interested in customizing it further. Specifically, I would like to include information about the number of datasets associated with each project. How can I go about customizing the display to include this dataset count?

**Comment by sriram-dsl** [comment]
Not exactly, but that's close. I'm interested in customizing the projects
page to display information about datasets, specifically details like no
datasets, closed datasets, and uploading datasets. I want to explore the
possibility of such customization on the web UI.

**Comment by ainoam** [comment]
@sriram-dsl You can find the UI source at its [GitHub repo](https://github.com/allegroai/clearml-web). I'd recommend joining the conversation at the [ClearML community slack](https://joinslack.clear.ml/) to consult with other users who have already customized the UI.
